### PR TITLE
Always activate the extension

### LINKED ISSUE #1991: When there are no files in the root directory, the Publisher Client does not load
When there are no files directly in the root folder opened in VSCode, the extension won't load.

To repro, you can either create a new folder with no files or delete the `.gitignore` from the `sample-content` folder.

This is thrown in Output > Window

> 2024-07-18 11:47:37.233 [error] Canceled: Canceled
	at new I (/Applications/Visual Studio Code.app/Contents/Resources/app/out/vs/workbench/api/node/extensionHostProcess.js:144:34990)
	at S.U (/Applications/Visual Studio Code.app/Contents/Resources/app/out/vs/workbench/api/node/extensionHostProcess.js:151:6023)
	at z.<computed>.F.charCodeAt.T.CharCode.DollarSign.z.<computed> (/Applications/Visual Studio Code.app/Contents/Resources/app/out/vs/workbench/api/node/extensionHostProcess.js:151:3107)
	at f.g (/Applications/Visual Studio Code.app/Contents/Resources/app/out/vs/workbench/api/node/extensionHostProcess.js:154:203861)
	at f.executeCommand (/Applications/Visual Studio Code.app/Contents/Resources/app/out/vs/workbench/api/node/extensionHostProcess.js:154:203299)
	at A.registerCommand.description (/Applications/Visual Studio Code.app/Contents/Resources/app/out/vs/workbench/api/node/extensionHostProcess.js:154:202623)
	at f.h (/Applications/Visual Studio Code.app/Contents/Resources/app/out/vs/workbench/api/node/extensionHostProcess.js:154:204400)
	at f.g (/Applications/Visual Studio Code.app/Contents/Resources/app/out/vs/workbench/api/node/extensionHostProcess.js:154:203392)
	at f.executeCommand (/Applications/Visual Studio Code.app/Contents/Resources/app/out/vs/workbench/api/node/extensionHostProcess.js:154:203299)
	at Object.executeCommand (/Applications/Visual Studio Code.app/Contents/Resources/app/out/vs/workbench/api/node/extensionHostProcess.js:164:23051)
	at n.O [as value] (/Applications/Visual Studio Code.app/Contents/Resources/app/extensions/git/dist/main.js:2:949496)
	at r.B (/Applications/Visual Studio Code.app/Contents/Resources/app/out/vs/workbench/api/node/extensionHostProcess.js:83:737)
	at r.C (/Applications/Visual Studio Code.app/Contents/Resources/app/out/vs/workbench/api/node/extensionHostProcess.js:83:807)
	at r.fire (/Applications/Visual Studio Code.app/Contents/Resources/app/out/vs/workbench/api/node/extensionHostProcess.js:83:1023)
	at Object.p [as dispose] (/Applications/Visual Studio Code.app/Contents/Resources/app/extensions/git/dist/main.js:2:968700)
	at /Applications/Visual Studio Code.app/Contents/Resources/app/extensions/git/dist/main.js:2:974738
	at Array.forEach (<anonymous>)
	at f.dispose (/Applications/Visual Studio Code.app/Contents/Resources/app/extensions/git/dist/main.js:2:974724)
	at c.<anonymous> (/Applications/Visual Studio Code.app/Contents/Resources/app/out/vs/workbench/api/node/extensionHostProcess.js:152:4048)
	at c.dispose (/Applications/Visual Studio Code.app/Contents/Resources/app/out/vs/workbench/api/node/extensionHostProcess.js:152:4145)
	at c.<anonymous> (/Applications/Visual Studio Code.app/Contents/Resources/app/extensions/git/dist/main.js:2:951753)
	at c.dispose (/Applications/Visual Studio Code.app/Contents/Resources/app/out/vs/workbench/api/node/extensionHostProcess.js:152:4145)
	at d (/Applications/Visual Studio Code.app/Contents/Resources/app/out/vs/workbench/api/node/extensionHostProcess.js:30:744)
	at m.eb (/Applications/Visual Studio Code.app/Contents/Resources/app/out/vs/workbench/api/node/extensionHostProcess.js:160:10039)
	at /Applications/Visual Studio Code.app/Contents/Resources/app/out/vs/workbench/api/node/extensionHostProcess.js:160:7915
	at Array.map (<anonymous>)
	at m.$ (/Applications/Visual Studio Code.app/Contents/Resources/app/out/vs/workbench/api/node/extensionHostProcess.js:160:7902)
	at m.terminate (/Applications/Visual Studio Code.app/Contents/Resources/app/out/vs/workbench/api/node/extensionHostProcess.js:160:8175)
	at d.terminate (/Applications/Visual Studio Code.app/Contents/Resources/app/out/vs/workbench/api/node/extensionHostProcess.js:164:1513)
	at i (/Applications/Visual Studio Code.app/Contents/Resources/app/out/vs/workbench/api/node/extensionHostProcess.js:177:10617)
	at MessagePortMain.<anonymous> (/Applications/Visual Studio Code.app/Contents/Resources/app/out/vs/workbench/api/node/extensionHostProcess.js:177:6809)
	at MessagePortMain.emit (node:events:514:28)
	at MessagePortMain._internalPort.emit (node:electron/js2c/utility_init:2:2285) setContext undefined

### PR DESCRIPTION
**Pull request by mmarchetti**
## Intent

This PR changes the activation events from `workspaceContains:*` to `workspaceContains:/`. The former requires a file to exist; the latter does not.

Fixes #1991 

## Type of Change

- - [x] Bug Fix 
- - [ ] New Feature 
- - [ ] Breaking Change 
- - [ ] Documentation 
- - [ ] Refactor 
- - [ ] Tooling 

## Directions for Reviewers

Open an empty folder; a folder containing only a folder; and a folder containing files. The extension icon should always show up.

### CODE REVIEW

File: extensions/vscode/package.json [hunk 1]
```diff
@@ -29,7 +29,7 @@
     "Other"
   ],
   "activationEvents": [
-    "workspaceContains:*"
+    "workspaceContains:/"
```
Comment (dotNomad) [inline extensions/vscode/package.json:32]: Should we change this to just `*` so the extension is always activated whenever VSCode starts up?

Comment (sagerb) [reply extensions/vscode/package.json:32]: That sounds like an excellent idea.

Comment (sagerb) [reply extensions/vscode/package.json:32]: I tried using the `*` and didn't see any visible improvement in load time, so I think my recommendation is to go with what Mike has currently in this PR.

**Review by sagerb — APPROVED** [review]
We can discuss changing this, but for now, I believe we should go with Microsoft's advice and use the setting that Mike has proposed.

File: extensions/vscode/package.json (hunk 1 above)
Comment (mmarchetti) [reply extensions/vscode/package.json:32]: Yes. Both will cause the extension to load when there's a workspace folder open; the difference is that `workspaceContains:/` will not launch the extension in empty workspaces.

Comment (dotNomad) [reply extensions/vscode/package.json:32]: Fantastic. Thank you both for digging into this.

**Pull request merged**